src/cheri/riscv-hybrid-integration.adoc

```diff
@@ -208,21 +208,28 @@ In {cheri_int_mode_name}, the capability authorizing the memory access is <<ddc>
 [#zicsr-section-default]
 === Changes to Zicsr Instructions
 
-{cheri_base_ext_name} extends all RISC-V CSRs which hold addresses, to hold capabilities.
-Therefore, such registers are extended to CLEN-bits in {cheri_base_ext_name} and a `c` suffix is added when referring to the entire CSR instead of the only the address field.
+//{cheri_base_ext_name} extends all RISC-V CSRs which hold addresses, to hold capabilities.
+//Therefore, such registers are extended to CLEN bits in {cheri_base_ext_name} and a `c` suffix is added when referring to the entire CSR instead of the only the address field.
 
-WARNING: This assumes that the Zcmt CSR is extended to be a capability when the base architecture is {cheri_base_ext_name}.
+//WARNING: This assumes that the Zcmt CSR is extended to be a capability when the base architecture is {cheri_base_ext_name}.
 
-For example, the `jvt` CSR is extended to `jvtc`, the `c` suffix indicating that it's a capability.
-It is referred to as `jvt` if only accessing the address or `jvtc` if accessing the whole capability.
+//For example, the `jvt` CSR is extended to `jvtc`, the `c` suffix indicating that it's a capability.
+//It is referred to as `jvt` if only accessing the address or `jvtc` if accessing the whole capability.
```

Comment:
I think it would make sense to remove lines we do not need anymore instead of commenting them out.

Reply:
agreed, and for the next two, I'll remove them